tomoxlending/order_processor.go

```diff
@@ -1036,14 +1039,17 @@ func (l *Lending) ProcessRepayLendingTrade(time uint64, lendingStateDB *lendings
 		err = lendingStateDB.RemoveLiquidationTime(lendingBook, lendingTradeId, lendingTrade.LiquidationTime)
 		if err != nil {
 			log.Debug("ProcessRepay RemoveLiquidationTime", "err", err, "lendingHash", lendingTrade.Hash, "trade", lendingstate.ToJSON(lendingTrade))
+			return nil,err
 		}
 		err = tradingstateDB.RemoveLiquidationPrice(tradingstate.GetTradingOrderBookHash(lendingTrade.CollateralToken, lendingTrade.LendingToken), lendingTrade.LiquidationPrice, lendingBook, lendingTradeId)
 		if err != nil {
 			log.Debug("ProcessRepay RemoveLiquidationPrice", "err", err)
+			return nil,err
 		}
 		lendingStateDB.CancelLendingTrade(lendingBook, lendingTradeId)
```

Comment:
check error from this function 
err = lendingStateDB.CancelLendingTrade(lendingBook, lendingTradeId)

Reply:
check error here:
https://github.com/tomochain/tomochain/pull/260/files#diff-bb070402da3cf4811b1d48bdf02391e6R997

https://github.com/tomochain/tomochain/pull/260/files#diff-bb070402da3cf4811b1d48bdf02391e6L1066

Reply:
updated